app/api/schema/settings.py

```diff
@@ -196,3 +196,16 @@ class Meta:
     smtp_password = fields.Str(allow_none=True)
     smtp_port = fields.Integer(allow_none=True)
     smtp_encryption = fields.Str(allow_none=True)  # Can be tls, ssl, none
+
+    # Admin Invoice Details
+    admin_billing_contact_name = fields.Str(allow_none=True)
+    admin_billing_phone = fields.Str(allow_none=True)
+    admin_billing_email = fields.Email(allow_none=True)
+    admin_billing_country = fields.Str(allow_none=True)
+    admin_billing_tax_info = fields.Str(allow_none=True)
+    admin_company = fields.Str(allow_none=True)
+    admin_billing_company_address = fields.Str(allow_none=True)
+    admin_billing_company_city = fields.Str(allow_none=True)
```

Comment:
admin_billing_city

Reply:
Outdated

Reply:
@mrsaicharan1 It'll automatically show outdated once you have done the change